components/ButtonBar/examples/ButtonBarCustomizedFontExample.m

```diff
@@ -30,22 +29,30 @@ - (id)init {
   if (self) {
     self.colorScheme =
         [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    self.typographyScheme = [[MDCTypographyScheme alloc] init];
-    
+    self.typographyScheme =
```

Comment:
Doesn't this not work if the designated initializers are called? Should we continue to add tech debt to our example code?

Reply:
Hrm I'm not sure I see what you're getting at here. Can you help me by clarifying what you mean by "Doesn't this not work if the designated initializers are called"? What would stop working?

Reply:
If `-initWithCoder:` is called, then the convenience initializer `-init` won't be called.

Reply:
Sorry I'm still not sure what you're referring to. To which object are you expecting initWithCoder to be called on?